examples/geoscience-aus-cat/projects_geoscience-aus-cat_assets_geomedians_LS8_preview.js

```diff
@@ -1,4 +1,4 @@
-var geomedian_ls5 = ee.ImageCollection('projects/geoscience-aus-cat/assets/geomedians_LS8');
```

Comment:
Neither the example nor the preview image have to show the whole dataset. Since this image has rough boundaries and gaps over the Eastern AU, you can simply zoom in to a prettier location. Not a blocker, but something you can consider for the future.